### PR TITLE
Update getting-started.md to include source

### PR DESCRIPTION
**Pull request by taylorreiter**
logging out and logging back in did not launch base conda environment, I need to source my .bashrc.

### CODE REVIEW

**Comment by SichongP** [comment]
You can also add the source code to .bash_profile so you don't have to run it every time you log in. It's been working for me. I think it's coz login shell doesn't run .bashrc.

File: getting-started.md [hunk 1]
```diff
@@ -37,7 +37,11 @@ bash Miniconda3-latest-Linux-x86_64.sh
 ```
 and then answer yes to all the questions!
 
-Log out and log back in again to activate the base conda environment.
+Log out and log back in again to activate the base conda environment. If this does not work, run:
+
+```
+source ~/.bashrc
```
Comment (bluegenes) [inline getting-started.md]: login shell runs ~/.bash_profile, not ~/.bashrc. We can copy the conda-init code to ~/.bash_profile with the following

cat ~/.bashrc >> ~/.bash_profile

**Review by bluegenes — CHANGES_REQUESTED** [review]
can we change the `source ~/.bashrc` to `cat ~/.bashrc >> ~/.bash_profile`? I think this is useful info for newcomers

**Comment by SichongP** [comment]
> can we change the source ~/.bashrc to cat ~/.bashrc >> ~/.bash_profile? I think this is useful info for newcomers

Is it better to add `source ~/.bashrc` to `~/.bash_profile`? This will make sure that every time the user logs in latest `.bashrc` file is sourced. And there is no need to update `.bash_profile` should `.bashrc` change again (with new app install, etc)

**Comment by ctb** [comment]
hi @bluegenes @luizirber @taylorreiter @SichongP  I think this is ready to merge now; review pls?

**Review by taylorreiter — COMMENTED** [review]
These changes look great to me! Thank you all!

File: getting-started.md (hunk 1 above)
Comment (taylorreiter) [reply getting-started.md]: @bluegenes, @ctb added `echo ~/.bashrc >> ~/.bash_profile`

**Pull request merged**